src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -172,6 +174,7 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
             setLevelOfEducation(toCopy.levelOfEducation);
             setExperience(toCopy.experience);
             setTags(toCopy.tags);
+            setDone(toCopy.done);
```

Comment:
I think since we are making unmark / mark its own command, we do not need to add it into `EditCommand.java`?